src/Test.UnitTests.Sarif/Core/ArtifactTests.cs

```diff
@@ -5,12 +5,13 @@
 using System.IO;
 using System.Text;
 using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif;
 using Microsoft.CodeAnalysis.Sarif.Writers;
 using Moq;
 using Newtonsoft.Json;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Sarif
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Core
```

Comment:
>Core [](start = 54, length = 4)

The unit tests are in a hodge-podge of namespaces, which makes it hard to find them in the VS test explorer. Since I let VS choose the namespace for the test classes I introduced in the PR, I took the opportunity to move several other test classes into the same default namespace.

Reply:
thanks this makes sense. i think the mismatched namespaces were a result of these files moving around at a point. thanks for fixing this, the test explorer bothered me too!

---
In reply to: [295397930](https://github.com/microsoft/sarif-sdk/pull/1537#discussion_r295397930) [](ancestors = 295397930)